add-on/chrome/bootstrap.js

```diff
@@ -180,21 +145,17 @@ var WindowListener = {
               // have resumed the tour as soon as the visitor joined if it was (and
               // the pref would have been set to false already.
               this.MozLoopService.resumeTour("waiting");
-              resolve();
+              resolve(null);
```

Comment:
So interestingly, I've just realised that openCallPanel doesn't actually resolve the promise in the non-resume case. Not sure if that means we're leaving things hanging, but I think we should tidy that up here.